src/deploymentmanager.ts

```diff
@@ -212,7 +213,11 @@ export class DeploymentManager implements IDeploymentManager {
                 }
 
                 if (answers.deploymentSku === 'local') {
-                    this.printEnvironmentVariables(deploymentProperties.outputs, storageEndpointSuffix);
+                    if (answers.setLocalEnv) {
```

Comment:
Should we not set the env on the machine by default?

Reply:
I though giving an option would be better.